lib/event-handlers.js

```diff
@@ -1,4 +1,8 @@
 'use strict';
+const _ = require('lodash');
+
+const storage = require('./storage');
+
 const props = require('../resources/properties');
```

Comment:
I like doing whitespace between external libraries and ones that are part of the project, so I'll just get rid of the whitespace between storage and props.